console/src/main/webapp/manager/app/assets/lang/de.json

```diff
@@ -1,175 +1,174 @@
   "org.address"           : "Adresse",
-  "org.save"              : "Enregistrer",
-  "org.updated"           : "Organisme modifié",
-  "org.deleted"           : "Organisme supprimé",
-  "org.created"           : "Organisme créé",
-  "org.error"             : "Erreur lors de la création de l’organisme",
-  "org.warning"           : "Attention, cette action est irréversible — ",
-  "org.delete"            : "Supprimer l’organisme",
-  "org.orglist"           : "Organismes",
-  "org.pendingmsg"        : "Organisme en attente ",
-  "org.confirm"           : "Confirmer",
-  "org.select"            : "Selectionner un organisme",
-  "org.membersCount"      : "Membres",
+  "org.save"              : "Speichern",
+  "org.updated"           : "Organisation aktualisiert",
+  "org.deleted"           : "Organisation gelöscht",
+  "org.created"           : "Organisation erstellt",
+  "org.error"             : "Fehler beim Erstellen der Organisation",
+  "org.warning"           : "Achtung, diese Aktion kann nicht rückgängig gemacht werden — ",
+  "org.delete"            : "Organisation löschen",
+  "org.orglist"           : "Organisationen",
+  "org.pendingmsg"        : "Ausstehende Organisationen ",
+  "org.confirm"           : "Bestätigen",
+  "org.select"            : "Organisation auswählen",
+  "org.membersCount"      : "Mitglieder",
 
-  "area.save"             : "Enregister la zone",
-  "area.updated"          : "Aire de compétence mise à jour",
-  "area.error"            : "Erreur d’enregistrement",
-  "area.selectByBBOX"     : "Sélection par BBOX",
-  "area.selection"        : "Sélection",
-  "area.area"             : "Zone(s)",
-  "area.all"              : "Tout",
-  "area.none"             : "Aucun",
-  "area.search"           : "Rechercher",
+  "area.save"             : "Bereich speichern",
+  "area.updated"          : "Bereich aktualisiert",
+  "area.error"            : "Fehler beim Speichern des Bereichs",
+  "area.selectByBBOX"     : "Rechteckauswahl",
+  "area.selection"        : "Auswählen",
+  "area.area"             : "Objekte",
+  "area.all"              : "Alle",
+  "area.none"             : "Keine",
+  "area.search"           : "Suchen",
 
-  "role.allusers"        : "Tous les utilisateurs",
-  "role.system"          : "Rôles geOrchestra",
-  "role.app"             : "Autres rôles",
-  "role.newuser"         : "Nouvel utilisateur",
-  "role.newrole"         : "Nouveau rôle",
-  "role.users"           : "utilisateurs",
-  "role.filter_users"    : "Filtrer",
-  "role.created"         : "Rôle créé",
-  "role.updated"         : "Rôle mis à jour",
-  "role.deleted"         : "Rôle supprimé",
-  "role.error"           : "Erreur de création du rôle",
-  "role.deleteError"     : "Erreur de suppression du rôle",
-  "role.search"          : "Chercher un rôle",
-  "role.all"             : "Tous",
-  "role.delete"          : "Supprimer",
+  "role.allusers"        : "Alle Benutzer",
+  "role.system"          : "geOrchestra-Benutzer",
```

Comment:
I'm really not sure for this one. It has to be understood as "static roles which are georchestra-specific, rather than freely created by administrators". So, I would translate it to "geOrchestra Rollen"